htdocs/web_portal/views/scope_help.php

```diff
@@ -44,26 +44,26 @@
               from all the child Services - a Service can't have a reserved tag that
               is not supported by its parent Site.</li>
             <li>For NGIs: If a Reserved tag is removed from an NGI, then the same tag is NOT
-              removed from all the child Sites - this is intentionally different from the Site->Service relationship.</li>
+              removed from all the child Sites - this is intentionally different from the Site&rarr;Service relationship.</li>
```

Comment:
Is this just a nicety or does the angle bracket cause problems?

Reply:
Both.
1. This is rich text, in this case a Unicode right arrow conveys the intended meaning better.
2. The `>` could be interpreted as the end of a tag, especially if there was a stray `<` further up.